src/routes/Home/components/CandidateSection/CandidateSection.js

```diff
@@ -15,22 +31,19 @@ class CandidateSection extends React.Component {
               <h1>Kandidat</h1>
             </div>
             <div className='row description'>
-              <p>Kandidat Pemilu {this.props.electionId}</p>
+              <p>Pasangan Kandidat di Pilkada Jawa Barat 2018</p>
             </div>
-            <CandidateList data={this.props.candidates} />
-            <Button fluid size='huge'
-              onClick={() => { history.push('/election/dkijakarta/candidate/1') }}>
-            Temp Candidate Detail Page</Button>
+            <Container>
+              <CandidateList data={this.state.data.candidate} />
+              <Button fluid size='huge'
+                onClick={() => { history.push('/election/dkijakarta/candidate/1') }}>
+                Temp Candidate Detail Page
+              </Button>
+            </Container>
           </div>
         </div>
       )} />
+
     )
   }
 }
-
-CandidateSection.propTypes = {
```

Comment:
Why did you remove this part? This part shouldn't be removed since it will check props that might be used by this components.